src/igvfd/schemas/tissue.json

```diff
@@ -77,6 +88,14 @@
                 "week"
             ]
         },
+        "ccf_id": {
+            "title": "Common coordinate framework identifier",
+            "description": "This identifier associates an entry from the HubMap Consortium for specimens, biological structures, and spatial positions.",
+            "comment": "This identifier will be later linked to: https://portal.hubmapconsortium.org/ccf-eui",
+            "type": "string",
+            "format": "uuid",
+            "nullable": true
```

Comment:
why should it be "nullable" true?

Reply:
either we have or we don't, i don't think we need nullable

Reply:
done